src/app/storage/types/json-patch.ts

```diff
@@ -0,0 +1,46 @@
+interface Patch {
```

Comment:
these are not being used currently, but I wanted to provide reference for `JSON-Patch` syntax, in case we choose to reuse any of this API's code elsewhere.